docs/guides/tutorials/cloudflare_workers.rst

```diff
@@ -0,0 +1,261 @@
+
+You can use this project as a reference: `EdgeDB Cloudflare Minimal Starter`_.
+
+Prerequisites
+-------------
+
+`Sign up for a Cloudflare account`_ to later deploy your worker.
+
+Ensure you have the following installed:
+
+- `Node.js`_
+- `EdgeDB CLI`_
+
+.. _Sign up for a Cloudflare account: https://dash.cloudflare.com/sign-up
+.. _Node.js: https://nodejs.org/en/
+.. _EdgeDB CLI: https://www.edgedb.com/docs/intro/cli
+
+Setup and Configuration
+-----------------------
+
+Initialize a New Cloudflare Worker Project
+===========================================
+
+Use the `create-cloudflare`_ package to create a new Cloudflare Worker project.
+
+.. _create-cloudflare: https://www.npmjs.com/package/create-cloudflare
+
+.. code-block:: bash
+
+    $ npm create cloudflare@latest # or pnpm, yarn, bun
+
+    # or
+    $ npx create-cloudflare@latest
+
+Answer the prompts to create a new project. Pick the *"Hello World" Worker*
+template to get started.
+   
+You'll be asked if you want to put your project on Cloudflare.
+If you say yes, you'll need to sign in (if you haven't already). 
+If you don't want to deploy up right away, just go to the project folder 
+you just made to start writing your code. When you're ready to put your 
+project on Cloudflare, you can run :code:`npx wrangler deploy` to push it.
+
+.. note:: Using Wrangler CLI
+
+    If you prefer using `Wrangler`_ to set up your worker, you can use the 
+    :code:`wrangler generate` command to create a new project.
+
+.. _Wrangler: https://developers.cloudflare.com/workers/cli-wrangler
+
+
+Configure EdgeDB
+=================
+
+You can use `EdgeDB Cloud`_ for a managed service or run EdgeDB locally.
+
+.. _`EdgeDB Cloud`: https://www.edgedb.com/cloud
+
+**Local EdgeDB Setup (Optional for EdgeDB Cloud Users)**
+
+If you're running EdgeDB locally, you can use the following command 
+to create a new instance:
+
+.. code-block:: bash
+
+    $ edgedb project init
+
+It creates an :code:`edgedb.toml` config file, and a schema file 
+:code:`dbschema/default.esdl`.
+
+It also spuns up an EdgeDB instance and “links” it to the current directory. 
+As long as you’re inside the project directory, all CLI commands will 
+be executed against this instance.
+
+You can run :code:`edgedb` in your terminal to open an 
+interactive REPL to your instance. 
+
+.. code-block:: bash
+
+    $ edgedb
+
+**Install The EdgeDB NPM package**
+
+.. code-block:: bash
+
+    $ npm install edgedb # or pnpm, yarn, bun
+
+**Extend The Default Schema (Optional)**
+
+You can extend the default schema, :code:`dbschema/default.esdl`, to define 
+your data model, and then try it out in the Cloudflare Worker code.
+
+Add new types to the schema file:
+
+.. code-block:: sdl
+
+    module default {
+      type Movie {
+        required property title -> str {
+          constraint exclusive;
+        };
+        multi link actors -> Person;
+      }
+
+      type Person {
+        required property name -> str;
+      }
```

Comment:
For new documentation, we've been preferring the newer EdgeDB SDL syntax [introduced in 3.0](https://www.edgedb.com/docs/changelog/3_x#simplified-sdl-syntax):

```suggestion
        required title: str {
          constraint exclusive;
        };
        multi actors: Person;
      }

      type Person {
        required name: str;
      }
```